Source/Core/Core/DSP/DSPTables.cpp

```diff
@@ -48,7 +48,22 @@ const std::array<DSPOPCTemplate, 214> s_opcodes =
   {"RETO",     0x02de, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return if overflow
   {"RET",      0x02df, 0xffff,    1, 0, {},                                                                                     false, true, true, false, false}, // unconditional return
 
-  {"RTI",      0x02ff, 0xffff,    1, 0, {},                                                                                     false, true, true, false, false}, // return from interrupt
+  {"RTIGE",    0x02f0, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return from interrupt if greater or equal
+  {"RTIL",     0x02f1, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return from interrupt if less
+  {"RTIG",     0x02f2, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return from interrupt if greater
+  {"RTILE",    0x02f3, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return from interrupt if less or equal
+  {"RTINZ",    0x02f4, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return from interrupt if not zero
+  {"RTIZ",     0x02f5, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return from interrupt if zero
+  {"RTINC",    0x02f6, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return from interrupt if not carry
+  {"RTIC",     0x02f7, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return from interrupt if carry
+  {"RTIx8",    0x02f8, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return from interrupt if TODO
+  {"RTIx9",    0x02f9, 0xffff,    1, 0, {},                                                                                     false, true, false, true, false}, // return from interrupt if TODO
```

Comment:
Chances are those two follow the remaining conditional opcode convention and check for above/below `s32` range (basically bit 4 of the status register).

Plus the following two for a yet unknown pair of checks.

Reply:
The "TODO" part for x8 and x9 is found with all of the instructions; I just copied that part from RET.  I can change it to "not over s32" and "over s32" respectively (for all of the conditional instructions), but I'll leave the xA and xB ones as `TODO`.

To be clear: The code currently assumes the same conditional behavior for RTIcc as all of the other conditional instructions (it uses `CheckCondition` for this); x8 through xB just don't have a label in DSPTables.

If you have a good summary for what this does:

https://github.com/dolphin-emu/dolphin/blob/a36855c9832847202f266e959584edd278363876/Source/Core/Core/DSP/Interpreter/DSPInterpreter.cpp#L256-L258

then I'd be happy to add a short description and a clearer mnemonic for all of them.

Reply:
Ah, tbf I didn't look at the others; just at the relation to the conditional instructions table in the docs.
In that case I'm all for consistency (and not just because I don't have any better name for them :P ) to leave them that way.

Reply:
I think I'll leave x8 and x9 alone for now too.  Based on 095f81e2f91844fe697f48d6bffd1f9476190f0e (and a quick check) they aren't used by any official uCode.  If further research indicates a clearer use for them, I'll create a separate PR.